master/buildbot/statistics/storage_backends/influxdb_client.py

```diff
@@ -58,17 +47,20 @@ def __init__(self, url, port, user, password, db, captures,
                                      self.password, self.db)
         self.inited = True
```

Comment:
nothing is really private in python... but I agree, self._inited remove the ambiguity of 'can we access from outside' ?